docs/antrea-network-policy.md

```diff
@@ -1433,6 +1474,101 @@ The following kubectl commands can be used to retrieve CG resources:
     kubectl get cg.crd.antrea.io
 ```
 
+## Group
+
+A Group (GRP) CRD is a another specification of how workloads are grouped together,
```

Comment:
"a another" -> "another"

or perhaps consider rephrasing as follows:

A Group CRD represents a different way for specifying how workloads are grouped together, and is conceptually similar to the ClusterGroup CRD. 
Users will be able to refer to Groups in Antrea NetworkPolicy objects instead of specifying pod and namespace selector every time.